test/test_alexa_integration_express.js

```diff
@@ -102,6 +113,24 @@ describe("Alexa", function() {
             expect(response.text).to.startWith('{"name":"testApp"')
           });
       });
+
+      it("returns debug schema", function() {
+        return request(testServer)
+          .get('/testApp?schema')
+          .expect(200).then(function(response) {
+            expect(response.headers['content-type']).to.equal('text/plain; charset=utf-8');
+            expect(response.text).to.include('"intent": "myIntent"')
+          });
+      });
+
+      it("returns debug utterances", function() {
+        return request(testServer)
+          .get('/testApp?utterances')
+          .expect(200).then(function(response) {
+            expect(response.headers['content-type']).to.equal('text/plain; charset=utf-8');
+            expect(response.text).to.include('my name is bob')
```

Comment:
Good point, even better updated to compare to the actual schema and utterances of the app.